notebook/static/base/js/namespace.js

```diff
@@ -1,10 +1,86 @@
 // Copyright (c) Jupyter Development Team.
 // Distributed under the terms of the Modified BSD License.
+//
+//
 
-var IPython = IPython || {};
-define([], function(){
+
+var Jupyter = Jupyter || {};
+
+var jprop = function(name, module_path){
+    Object.defineProperty(Jupyter, name, {
+      get: function() { 
+          console.warn('accessing `'+name+'` is deprecated. Use require("'+module_path+'")');
```

Comment:
You surround `name` in backticks, so you should surround `require("...")` in backticks too.